src/auth/index.js

```diff
@@ -208,19 +213,28 @@ export default class Auth {
     return this.storage.get(authCommon.REFRESH_TOKEN_KEY);
   }
 
-  set(json) {
+  setAuthType(authType) {
+    return new Promise((resolve) =>
+      resolve(this.storage.set(authCommon.USER_LOGGED_IN_PT_KEY, authType))
+    );
+  }
+
+  set(json, authType) {
```

Comment:
you could use a default value `authType = ''` and then not do the `!authType` check in the promise that @edaniels commented on

Reply:
done